src/pages/Group/GroupSpendersCard/GroupSpendersCard.tsx

```diff
@@ -26,7 +26,7 @@ const GroupSpendersCard: FC<IGroupSpendersCardProps> = ({data, isLoading, isErro
         groupId: string,
         memberId: string
     }>();
-    
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency)
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency)